prov/shm/src/smr_init.c

```diff
@@ -61,14 +61,38 @@ static void smr_resolve_addr(const char *node, const char *service,
 	*addrlen = strlen(*addr);
 }
 
+static int smr_get_ptrace_scope(void)
+{
+	FILE *file;
+	int scope, ret;
+
+	scope = 0;
```

Comment:
Maybe initialize to ENOSYS/ENOTSUPP and only return 0 if we can verify that scope is 0?

Reply:
Well the file doesn't always exist. If it doesn't, I believe it defaults to 0. At least it does on our rhel systems. Perhaps there's another way to verify. I wasn't able to find another way.

Reply:
All righty then.  :)